From 949b24a42f8bc0df899712ab03bbc7fa2212b4e4 Mon Sep 17 00:00:00 2001
From: Andreas Schwab <schwab@suse.de>
Date: Tue, 9 Apr 2019 10:52:53 +0200
Subject: [PATCH] Define the riscv architecture feature

---
 src/tools/features/architecture-feature.jam | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tools/build/src/tools/features/architecture-feature.jam b/tools/build/src/tools/features/architecture-feature.jam
index b8f7fbcbdf..54f9a8f4a6 100644
--- a/tools/build/src/tools/features/architecture-feature.jam
+++ b/tools/build/src/tools/features/architecture-feature.jam
@@ -30,6 +30,9 @@ feature.feature architecture
         # Advanced RISC Machines
         arm
 
+        # RISC-V
+        riscv
+
         # Combined architectures for platforms/toolsets that support building for
         # multiple architectures at once. "combined" would be the default multi-arch
         # for the toolset.
