diff --git a/libs/context/src/asm/jump_arm64_aapcs_elf_gas.S b/libs/context/src/asm/jump_arm64_aapcs_elf_gas.S
index 7f012e6..f14cafc 100644
--- a/libs/context/src/asm/jump_arm64_aapcs_elf_gas.S
+++ b/libs/context/src/asm/jump_arm64_aapcs_elf_gas.S
@@ -37,7 +37,6 @@
  *                                                     *
  *******************************************************/
 
-.cpu    generic+fp+simd
 .text
 .align  2
 .global jump_fcontext
diff --git a/libs/context/src/asm/make_arm64_aapcs_elf_gas.S b/libs/context/src/asm/make_arm64_aapcs_elf_gas.S
index ba8984b..cb1f51a 100644
--- a/libs/context/src/asm/make_arm64_aapcs_elf_gas.S
+++ b/libs/context/src/asm/make_arm64_aapcs_elf_gas.S
@@ -37,7 +37,6 @@
  *                                                     *
  *******************************************************/
 
-.cpu    generic+fp+simd
 .text
 .align  2
 .global make_fcontext
diff --git a/libs/context/src/asm/ontop_arm64_aapcs_elf_gas.S b/libs/context/src/asm/ontop_arm64_aapcs_elf_gas.S
index f33599a..72b03eb 100644
--- a/libs/context/src/asm/ontop_arm64_aapcs_elf_gas.S
+++ b/libs/context/src/asm/ontop_arm64_aapcs_elf_gas.S
@@ -37,7 +37,6 @@
  *                                                     *
  *******************************************************/
 
-.cpu    generic+fp+simd
 .text
 .align  2
 .global ontop_fcontext
