diff -up boost_1_83_0/libs/url/src/segments_view.cpp.1~ boost_1_83_0/libs/url/src/segments_view.cpp
--- boost_1_83_0/libs/url/src/segments_view.cpp.1~	2023-08-11 20:45:15.450460837 +0200
+++ boost_1_83_0/libs/url/src/segments_view.cpp	2023-08-11 20:46:14.542902137 +0200
@@ -28,10 +28,15 @@ segments_view(
 segments_view::
 segments_view(
     core::string_view s)
+#if 0
     : segments_view(
         parse_path(s).value(
             BOOST_URL_POS))
+#endif
 {
+#if 1
+    *this = parse_path(s).value(BOOST_URL_POS);
+#endif
 }
 
 } // urls
