diff -up boost_1_70_0/tools/build/src/tools/gcc.jam.0019~ boost_1_70_0/tools/build/src/tools/gcc.jam
--- boost_1_70_0/tools/build/src/tools/gcc.jam.0019~	2019-04-09 21:36:57.000000000 +0200
+++ boost_1_70_0/tools/build/src/tools/gcc.jam	2019-04-12 14:23:10.519125908 +0200
@@ -569,7 +569,7 @@ rule compile.fortran ( targets * : sourc
 
 actions compile.c++ bind PCH_FILE
 {
-    "$(CONFIG_COMMAND)" $(LANG) -ftemplate-depth-$(TEMPLATE_DEPTH) $(OPTIONS) $(USER_OPTIONS) -D$(DEFINES) -I"$(PCH_FILE:D)" -I"$(INCLUDES)" -c -o "$(<:W)" "$(>:W)"
+    "$(CONFIG_COMMAND)" $(LANG) $(OPTIONS) $(USER_OPTIONS) -D$(DEFINES) -I"$(PCH_FILE:D)" -I"$(INCLUDES)" -c -o "$(<:W)" "$(>:W)"
 }
 
 actions compile.c bind PCH_FILE
@@ -579,7 +579,7 @@ actions compile.c bind PCH_FILE
 
 actions compile.c++.preprocess bind PCH_FILE
 {
-    "$(CONFIG_COMMAND)" $(LANG) -ftemplate-depth-$(TEMPLATE_DEPTH) $(OPTIONS) $(USER_OPTIONS) -D$(DEFINES) -I"$(PCH_FILE:D)" -I"$(INCLUDES)" "$(>:W)" -E >"$(<:W)"
+    "$(CONFIG_COMMAND)" $(LANG) $(OPTIONS) $(USER_OPTIONS) -D$(DEFINES) -I"$(PCH_FILE:D)" -I"$(INCLUDES)" "$(>:W)" -E >"$(<:W)"
 }
 
 actions compile.c.preprocess bind PCH_FILE
@@ -702,13 +702,13 @@ actions compile.c.pch
 ###
 
 # Declare flags and action for compilation.
-toolset.flags gcc.compile OPTIONS <optimization>off   : -O0 ;
-toolset.flags gcc.compile OPTIONS <optimization>speed : -O3 ;
-toolset.flags gcc.compile OPTIONS <optimization>space : -Os ;
-
-toolset.flags gcc.compile OPTIONS <inlining>off  : -fno-inline ;
-toolset.flags gcc.compile OPTIONS <inlining>on   : -Wno-inline ;
-toolset.flags gcc.compile OPTIONS <inlining>full : -finline-functions -Wno-inline ;
+toolset.flags gcc.compile OPTIONS <optimization>off   : ;
+toolset.flags gcc.compile OPTIONS <optimization>speed : ;
+toolset.flags gcc.compile OPTIONS <optimization>space : ;
+
+toolset.flags gcc.compile OPTIONS <inlining>off  : ;
+toolset.flags gcc.compile OPTIONS <inlining>on   : ;
+toolset.flags gcc.compile OPTIONS <inlining>full : ;
 
 toolset.flags gcc.compile OPTIONS <warnings>off : -w ;
 toolset.flags gcc.compile OPTIONS <warnings>on  : -Wall ;
