diff -up boost_1_68_0/boost/atomic/detail/ops_gcc_x86_dcas.hpp.0006~ boost_1_68_0/boost/atomic/detail/ops_gcc_x86_dcas.hpp
--- boost_1_68_0/boost/atomic/detail/ops_gcc_x86_dcas.hpp.0006~	2018-08-01 22:50:46.000000000 +0200
+++ boost_1_68_0/boost/atomic/detail/ops_gcc_x86_dcas.hpp	2018-08-10 01:08:54.617706383 +0200
@@ -119,7 +119,7 @@ struct gcc_dcas_x86
         }
     }
 
-    static BOOST_FORCEINLINE storage_type load(storage_type const volatile& storage, memory_order) BOOST_NOEXCEPT
+    static BOOST_FORCEINLINE storage_type load(storage_type volatile& storage, memory_order) BOOST_NOEXCEPT
     {
         storage_type value;
 
@@ -404,7 +404,7 @@ struct gcc_dcas_x86_64
         );
     }
 
-    static BOOST_FORCEINLINE storage_type load(storage_type const volatile& storage, memory_order) BOOST_NOEXCEPT
+    static BOOST_FORCEINLINE storage_type load(storage_type volatile& storage, memory_order) BOOST_NOEXCEPT
     {
         // Note that despite const qualification cmpxchg16b below may issue a store to the storage. The storage value
         // will not change, but this prevents the storage to reside in read-only memory.
