diff -up boost_1_68_0/boost/atomic/detail/ops_gcc_x86_dcas.hpp.omv~ boost_1_68_0/boost/atomic/detail/ops_gcc_x86_dcas.hpp
--- boost_1_68_0/boost/atomic/detail/ops_gcc_x86_dcas.hpp.omv~	2018-07-28 18:45:29.396518945 +0200
+++ boost_1_68_0/boost/atomic/detail/ops_gcc_x86_dcas.hpp	2018-07-28 19:37:34.889519074 +0200
@@ -119,7 +119,7 @@ struct gcc_dcas_x86
         }
     }
 
-    static BOOST_FORCEINLINE storage_type load(storage_type const volatile& storage, memory_order) BOOST_NOEXCEPT
+    static BOOST_FORCEINLINE storage_type load(storage_type volatile& storage, memory_order) BOOST_NOEXCEPT
     {
         storage_type value;
 
@@ -399,7 +399,7 @@ struct gcc_dcas_x86_64
         );
     }
 
-    static BOOST_FORCEINLINE storage_type load(storage_type const volatile& storage, memory_order) BOOST_NOEXCEPT
+    static BOOST_FORCEINLINE storage_type load(storage_type volatile& storage, memory_order) BOOST_NOEXCEPT
     {
 #if defined(__clang__)
 
