diff -up boost_1_82_0/boost/url/impl/segments_view.ipp.omv~ boost_1_82_0/boost/url/impl/segments_view.ipp
--- boost_1_82_0/boost/url/impl/segments_view.ipp.omv~	2023-04-14 21:41:11.131126628 +0200
+++ boost_1_82_0/boost/url/impl/segments_view.ipp	2023-04-14 21:41:45.332429477 +0200
@@ -27,10 +27,15 @@ segments_view(
 segments_view::
 segments_view(
     string_view s)
+#if 0
     : segments_view(
         parse_path(s).value(
             BOOST_URL_POS))
+#endif
 {
+#if 1
+    *this = parse_path(s).value(BOOST_URL_POS);
+#endif
 }
 
 } // urls
