--- boost_1_60_0/libs/context/src/asm/jump_arm64_aapcs_elf_gas.S.omv~	2016-03-21 11:50:38.017571560 +0100
+++ boost_1_60_0/libs/context/src/asm/jump_arm64_aapcs_elf_gas.S	2016-03-21 11:50:42.517563754 +0100
@@ -51,7 +51,6 @@
  *                                                     *
  *******************************************************/
 
-.cpu    generic+fp+simd
 .text
 .align  2
 .global jump_fcontext
--- boost_1_60_0/libs/context/src/asm/make_arm64_aapcs_elf_gas.S.omv~	2016-03-21 11:50:46.577556589 +0100
+++ boost_1_60_0/libs/context/src/asm/make_arm64_aapcs_elf_gas.S	2016-03-21 11:50:52.707545546 +0100
@@ -51,7 +51,6 @@
  *                                                     *
  *******************************************************/
 
-.cpu    generic+fp+simd
 .text
 .align  2
 .global make_fcontext
