diff -up boost_1_80_0/libs/locale/src/icu/collator.cpp.10~ boost_1_80_0/libs/locale/src/icu/collator.cpp
--- boost_1_80_0/libs/locale/src/icu/collator.cpp.10~	2022-07-18 17:13:13.973986757 +0200
+++ boost_1_80_0/libs/locale/src/icu/collator.cpp	2022-07-18 17:17:10.442425112 +0200
@@ -93,7 +93,7 @@ namespace boost {
                     std::vector<uint8_t> tmp;
                     tmp.resize(str.length() + 1u);
                     icu::Collator *collate = get_collator(level);
-                    const int len = collate->getSortKey(str,&tmp[0],tmp.size());
+                    const int len = collate->getSortKey(str,tmp.empty()?nullptr:&tmp[0],tmp.size());
                     if(len > int(tmp.size())) {
                         tmp.resize(len);
                         collate->getSortKey(str,&tmp[0],tmp.size());
